packages/terra-form-field/CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ Unreleased
 ### Changed
 * Made required updates to consume terra-toolkit v5 and terra-dev-site v5
 
+3.10.0 - (May 1, 2019)
```

Comment:
Probably don't want to duplicate the date in the change log.

Reply:
Ope, updated: https://github.com/cerner/terra-core/pull/2398/commits/eedb05556d54614e8a3acc05888e4973a7be2a06